src/controllers/notifications.js

```diff
@@ -86,16 +90,47 @@ exports.post = function * postNotification () {
   try {
     yield model.processNotification(notification, this.ledgers, this.config)
   } catch (e) {
-    if (!(e instanceof UnacceptableExpiryError)) {
+    if (
+      (e instanceof AssetsNotTradedError) ||
+      (e instanceof NoRelatedDestinationDebitError) ||
+      (e instanceof NoRelatedSourceCreditError) ||
+      (e instanceof UnacceptableConditionsError) ||
+      (e instanceof UnacceptableExpiryError) ||
+      (e instanceof UnacceptableRateError) ||
+      (e instanceof UnrelatedNotificationError)
+    ) {
+      // Certain exceptions indicate an error in the transfer memo, rather than
```

Comment:
After reading about it "202 Accepted" seems to be more for requests that haven't been processed yet, which isn't quite the case here. If we wanted to return 202, we should probably return it always and actually do the processing after the request. I'll leave the decision about switching to 202 for a future PR.